benchmark/opperf/nd_operations/random_sampling_operators.py

```diff
@@ -19,12 +19,12 @@
 1. Operators are automatically fetched from MXNet operator registry.
 2. Default Inputs are generated. See rules/default_params.py. You can override the default values.
 
-Below 16 random sampling Operators are covered:
+Below 19 random sampling Operators are covered:
 
 ['random_exponential', 'random_gamma', 'random_generalized_negative_binomial', 'random_negative_binomial',
 'random_normal', 'random_poisson', 'random_randint', 'random_uniform', 'sample_exponential', 'sample_gamma',
 'sample_generalized_negative_binomial', 'sample_multinomial', 'sample_negative_binomial', 'sample_normal',
-'sample_poisson', 'sample_uniform']
+'sample_poisson', 'sample_uniform', 'GridGenerator', 'sample_multinomial', 'BilinearSampler']
```

Comment:
already sample_multinomial exists by mistake! don't add again :p

Reply:
I added it with custom data to make it run more efficiently during perf tests. Would you prefer I remove it?

Reply:
Good catch 👍